backend/src/main/java/com/benine/backend/http/CameraStreamHandler.java

```diff
@@ -0,0 +1,88 @@
+package com.benine.backend.http;
+
+import com.benine.backend.LogEvent;
+import com.benine.backend.ServerController;
+import com.benine.backend.video.MJPEGStreamReader;
+import com.benine.backend.video.StreamDistributer;
+import com.benine.backend.video.StreamNotAvailableException;
+import com.benine.backend.video.StreamReader;
+import org.eclipse.jetty.server.Request;
+
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.PipedInputStream;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Class responsible for handling the /camera/ route.
+ */
+public class CameraStreamHandler extends CameraRequestHandler {
+
+  @Override
+  public void handle(String s, Request request, HttpServletRequest req, HttpServletResponse res)
+          throws IOException, ServletException {
+
+    int camID = getCameraId(request);
+
+    StreamReader streamReader = null;
+    try {
+      streamReader = ServerController.getInstance().getStreamController().getStreamReader(camID);
+    } catch (StreamNotAvailableException e) {
+      e.printStackTrace();
```

Comment:
Please log an exception here like "getLogger.log([String:description], e);"  this also automatically logs the stacktrace (and is thus stored in the logs).